nwb_conversion_tools/utils/basenwbephyswriter.py

```diff
@@ -110,6 +114,40 @@ def _get_unit_waveforms_templates(self, unit_id, mode="mean"):
         """
         pass
 
+    def get_default_nwbfile_metadata(self):
+        """
+        Returns structure with defaulted metadata values required for a NWBFile.
+        """
+        metadata = dict(
+            NWBFile=dict(
+                session_description="no description",
+                session_start_time=datetime(1970, 1, 1),
+                identifier=str(uuid.uuid4()),
+            ),
+            Ecephys=dict(
+                Device=[dict(name="Device", description="Ecephys probe. Automatically generated.")],
+                Electrodes=[],
+                ElectrodeGroup=[],
```

Comment:
I'd rather not have empty fields

Reply:
reverted https://github.com/catalystneuro/nwb-conversion-tools/pull/230/commits/6fec4f5ef7268be30bc7e8d78561b8fed70ce56b